python/models.md

```diff
@@ -1,9 +1,31 @@
 # Model style
 
+General form:
+
+```
+class StyleGuide(models.Model):
+
```

Comment:
Fair enough. I'm open to either just wondered what the reasoning behind it was 👍